scripts/tokenomics-emulation/generate-chain-spec.ts

```diff
@@ -49,7 +69,7 @@ class ChainSpecGenerator {
 
         const treasuryStake = 10000;
 
-        const rootAccountBalance = config.network.total_supply - aliceBalance - totalGenesisValidatorsStake - COUNCIL_MULTIPLIER * totalGenesisCouncilsStake - treasuryStake;
+        const rootAccountBalance = config.network.total_supply - aliceBalance - totalGenesisValidatorsStake - totalNominators - totalValidators - COUNCIL_MULTIPLIER * totalGenesisCouncilsStake - treasuryStake;
```

Comment:
add `Stake` suffix to new variables here.

Reply:
added the suffix